lib/rspec/core/formatters/message_formatter.rb

```diff
@@ -0,0 +1,28 @@
+module RSpec
+  module Core
+    module Formatters
+      # @api private
+      # Formatter for providing message output as a fallback when no other
+      # profiler implements #message
+      class MessageFormatter
+        Formatters.register self, :message
+
+        def initialize(output)
+          @output = output
+        end
+
+        # @private
+        attr_reader :output
+
+        # @api public
+        #
+        # Used by the reporter to send messages to the output stream.
+        #
+        # @param notification [MessageNotification] containing message
+        def message(notification)
+          output.puts notification.message
```

Comment:
The build is failing because this line is not covered by a test.  (`script/rspec_with_simplecov` will check coverage).